drivers/timer/ite_it8xxx2_timer.c

```diff
@@ -73,6 +73,13 @@ const int32_t z_sys_timer_irq_for_test = DT_IRQ_BY_IDX(DT_NODELABEL(timer), 5, i
 #define EVEN_TIMER_MAX_CNT_SYS_TICK	(EVENT_TIMER_MAX_CNT \
 					/ HW_CNT_PER_SYS_TICK)
 
+/* Timer tick threshold to prevent SoC from entering idle mode.
+ * Calculated as 150µs converted to timer ticks using the formula:
+ *   ticks = us * timer_clk_src / 1000000
+ * where (event/free run timers)timer_clk_src is fixed at 32768Hz
+ */
+#define IDLE_BLOCK_TIMER_TICKS DIV_ROUND_UP(150 * 32768, 1000000)
```

Comment:
Should this use `CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC`?
```suggestion
#define IDLE_BLOCK_TIMER_TICKS DIV_ROUND_UP(150 * CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC, 1000000)
```